README.md

```diff
@@ -1 +1,188 @@
-# sti-python
+Python for OpenShift - Docker images
+========================================
+
+This repository contains sources of the images for building various versions
+of Python applications as reproducible Docker images using
+[source-to-image](https://github.com/openshift/source-to-image).
+User can choose between RHEL and CentOS based builder images.
+The resulting image can be run using [Docker](http://docker.io).
+
+
+Versions
+---------------
+Python versions currently supported are:
+* python-3.3
+
+RHEL versions currently supported are:
+* RHEL7
+
+CentOS versions currently supported are:
+* CentOS7
+
+
+Installation
+---------------
+To build Python image, choose between CentOS or RHEL based image:
+*  **RHEL based image**
+
+    To build a rhel-based python-3.3 image, you need to run the build on properly
```

Comment:
It's there, but wrapped in the next line :wink: I'll move it into this line then.